controllers/allcontrollers_test.go

```diff
@@ -128,3 +128,17 @@ routingTemplates:
 		return false
 	}, time.Second*2, time.Millisecond*100)
 }
+
+func TestGetVolumeUsage(t *testing.T) {
+	// GetVolumeUsage is based off of statfs which analyzes the volume, not the directory
+	// Creating a temporary directory will not change anything
+	path, err := os.Getwd()
```

Comment:
Creating a temporary directory will not do anything here. Unless, we want to create a volume mount? I am not sure if we even have permission to do that.